docs/DeveloperGuide.adoc

```diff
@@ -74,9 +79,9 @@ Optionally, you can follow the <<UsingCheckstyle#, UsingCheckstyle.adoc>> docume
 
 ==== Updating documentation to match your fork
 
-After forking the repo, the documentation will still have the SE-EDU branding and refer to the `se-edu/addressbook-level4` repo.
+After forking the repo, the documentation will still have the SE-EDU branding and refer to the `se-edu/TravelBuddy-level4` repo.
 
-If you plan to develop this fork as a separate product (i.e. instead of contributing to `se-edu/addressbook-level4`), you should do the following:
+If you plan to develop this fork as a separate product (i.e. instead of contributing to `se-edu/TravelBuddy-level4`), you should do the following:
```

Comment:
Perhaps the section on "updating documentation to match your fork" can be removed as it is not relevant to TravelBuddy. Do note that "se-edu/TravelBuddy-level4" does not exist.

Reply:
Removed the section